src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java

```diff
@@ -128,6 +128,16 @@
             optional = true)
     private Double vqsLodINDELThreshold = null;
 
+    /**
+    * If this flag is enabled, sites that have been marked as filtered (i.e. have anything other than `.` or `PASS`
+    * in the FILTER field) will be excluded from the output.
+    */
+    @Argument(
+            fullName="exclude-filtered",
+            doc="Don't include filtered sites in the final jointVCF",
+            optional=true)
+    private boolean XLfiltered = false;
```

Comment:
typically Java variables start with lower case, (and XL looks like extra large to me!).  How about `excludeFilteredSites` just to be super clear?

Reply:
yeah---XL did feel weird---but I wanted to lift it directly from SelectVariants. Is it better to be clear or match the already existing pattern?